static/css/footer.less

```diff
@@ -24,121 +24,53 @@
     background: @gray-5;
 
     &_share-icon-list {
-        margin-bottom: unit(30px / @base-font-size-px, em);
+        margin-top: unit(30px / @base-font-size-px, em);
```

Comment:
I think you could use `@grid_gutter-width` (https://github.com/cfpb/cfgov-refresh/blob/flapjack/static/css/cf-theme-overrides.less#L138) in place of `30px`.